### PR TITLE
feat: Add Weaverbird telemetry defenitions

### PR DESCRIPTION
**Pull request by dianaabv16**
## Problem
Weaverbird telemetry should be added.
Context: Weaverbird is a part of aws Toolkit. 
## License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: telemetry/definitions/commonDefinitions.json [hunk 1]
```diff
@@ -855,6 +855,71 @@
             "name": "userChoice",
             "type": "string",
             "description": "User selection from a predefined menu (not user-provided input)" 
+        },
+        {
+            "name": "assignClickCount",
```
Comment (justinmk3) [inline telemetry/definitions/commonDefinitions.json:860]: if these new types are not generically useful for other components, they need a prefix. look at other types for example

File: telemetry/definitions/commonDefinitions.json [hunk 2]
```diff
@@ -2544,6 +2609,51 @@
             ],
             "passive": true
         },
+        {
+            "name": "weaverbird_serviceInvocation",
+            "description": "Client side invocation of the weavervird service for solving software engineering tasks",
```
Comment (jpinkney-aws) [inline telemetry/definitions/commonDefinitions.json:2614]:
```suggestion
            "description": "Client side invocation of the weaverbird service for solving software engineering tasks",
```

File: telemetry/definitions/commonDefinitions.json (hunk 1 above)
Comment (dianaabv16) [reply telemetry/definitions/commonDefinitions.json]: addressed

File: telemetry/definitions/commonDefinitions.json [hunk 3]
```diff
@@ -855,6 +855,71 @@
             "name": "userChoice",
             "type": "string",
             "description": "User selection from a predefined menu (not user-provided input)" 
+        },
+        {
+            "name": "awsqAssignClickCount",
```
Comment (jpinkney-aws) [inline telemetry/definitions/commonDefinitions.json:860]: @justinmk3 Is there a defined convention on these names? We're following how codewhisperer formats their names (camal case), but other teams like codecatalyst seem to prefix with `codecatalyst_${name}`

Comment (justinmk3) [reply telemetry/definitions/commonDefinitions.json:860]: The codecatalyst names were an accident, I think. Probably best to continue using `topicCamelCase` for types. Whereas metric names use `topic_camelCase`.

Comment (dianaabv16) [reply telemetry/definitions/commonDefinitions.json:860]: @justinmk3  can I stil awsq_ or is awsQ_ is expected?

File: telemetry/definitions/commonDefinitions.json [hunk 4]
```diff
@@ -2544,6 +2609,51 @@
...
+                    "type": "awsqApproachIterationCount"
+                },
+                {
+                    "type": "awsqIsRegeneratedCode"
+                },
+                {
+                    "type": "awsqFilesReviewedCount"
+                },
+                {
+                    "type": "awsqIsAcceptedCodeChanges"
```
Comment (jpinkney-aws) [inline telemetry/definitions/commonDefinitions.json:2635]: For this one and isApproachAccepted do we need the "Is" part?

Comment (dianaabv16) [reply telemetry/definitions/commonDefinitions.json:2635]: booleans in the file start with `is` or `has` don't think there is a rule for that.
@justinmk3 wdyt?

File: telemetry/definitions/commonDefinitions.json (hunk 3 above)
Comment (justinmk3) [reply telemetry/definitions/commonDefinitions.json:860]: `awsq_`. sorry for confusion, updated above comment.

**Review by jpinkney-aws — APPROVED** [review]

File: telemetry/definitions/commonDefinitions.json [hunk 5]
```diff
@@ -855,6 +855,71 @@
             "name": "userChoice",
             "type": "string",
             "description": "User selection from a predefined menu (not user-provided input)" 
+        },
+        {
+            "name": "awsqAssignClickCount",
+            "type": "int",
+            "description": "Number of times users have triggered /assign"
```
Comment (dhruvigajjar) [inline telemetry/definitions/commonDefinitions.json:862]: is there a time frame or session under which this `awsqAssignClickCount` will be updated?

**Pull request merged**